### PR TITLE
Change start date so 2023 is removed from the list of teacher training years in adviser funnel

### PR DESCRIPTION
**Pull request by plisovin**
### Trello card

[Trello-4965](https://trello.com/c/u5vUNbgK/4965-ensure-that-2023-is-removed-from-the-list-of-teacher-training-years-in-adviser-funnel)

### Context

The adviser funnel asks users when they want to start their teacher training. Once the recruitment cycle is closed at the beginning of September, we need to ensure that the list does not contain the current year (2023).

### CODE REVIEW

File: spec/models/teacher_training_adviser/steps/start_teacher_training_spec.rb [hunk 1]
```diff
@@ -159,7 +159,7 @@
 
     it "returns current calendar year + 2 years if first year and before 7th September" do
```
Comment (gemmadallmandfe) [inline spec/models/teacher_training_adviser/steps/start_teacher_training_spec.rb:160]: does the comment need updating - still says `7th September`?

File: spec/models/teacher_training_adviser/steps/start_teacher_training_spec.rb [hunk 2]
```diff
@@ -173,7 +173,7 @@
 
     it "returns current calendar year + 1 years if second year and before 7th September" do
```
Comment (gemmadallmandfe) [inline spec/models/teacher_training_adviser/steps/start_teacher_training_spec.rb:174]: does the comment need updating - still says `7th September`?

File: spec/models/teacher_training_adviser/steps/start_teacher_training_spec.rb [hunk 3]
```diff
@@ -187,7 +187,7 @@
 
     it "returns current calendar year + 1 years if other and before 7th September" do
```
Comment (gemmadallmandfe) [inline spec/models/teacher_training_adviser/steps/start_teacher_training_spec.rb:188]: does the comment need updating - still says `7th September`?

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for this to be merged

File: spec/models/teacher_training_adviser/steps/start_teacher_training_spec.rb [hunk 4]
```diff
@@ -157,59 +157,59 @@
       wizardstore["degree_options"] = TeacherTrainingAdviser::Steps::HaveADegree::DEGREE_OPTIONS[:studying]
     end
 
-    it "returns current calendar year + 2 years if first year and before 7th September" do
+    it "returns current calendar year + 2 years if first year and before 4th September" do
       wizardstore["degree_status_id"] = TeacherTrainingAdviser::Steps::StageOfDegree::NOT_FINAL_YEAR[:first_year]
-      travel_to(Date.new(2022, 9, 6)) do
+      travel_to(Date.new(2022, 9, 2)) do
```
Comment (asatwal) [inline spec/models/teacher_training_adviser/steps/start_teacher_training_spec.rb:162]: Same as above, should this be day 3 ?

File: spec/models/teacher_training_adviser/steps/start_teacher_training_spec.rb [hunk 5]
```diff
@@ -157,59 +157,59 @@
...
+      travel_to(Date.new(2022, 9, 4)) do
         expect(instance.inferred_year_id).to eq(12_923)
       end
     end
 
-    it "returns current calendar year + 1 years if second year and before 7th September" do
+    it "returns current calendar year + 1 years if second year and before 4th September" do
       wizardstore["degree_status_id"] = TeacherTrainingAdviser::Steps::StageOfDegree::NOT_FINAL_YEAR[:second_year]
-      travel_to(Date.new(2022, 9, 6)) do
+      travel_to(Date.new(2022, 9, 2)) do
```
Comment (asatwal) [inline spec/models/teacher_training_adviser/steps/start_teacher_training_spec.rb:176]: Same as above, should this be day 3 ?

File: spec/models/teacher_training_adviser/steps/start_teacher_training_spec.rb [hunk 6]
```diff
@@ -157,59 +157,59 @@
...
+      travel_to(Date.new(2022, 9, 4)) do
         expect(instance.inferred_year_id).to eq(12_922)
       end
     end
 
-    it "returns current calendar year + 1 years if other and before 7th September" do
+    it "returns current calendar year + 1 years if other and before 4th September" do
       wizardstore["degree_status_id"] = TeacherTrainingAdviser::Steps::StageOfDegree::NOT_FINAL_YEAR[:other]
-      travel_to(Date.new(2022, 9, 6)) do
+      travel_to(Date.new(2022, 9, 2)) do
```
Comment (asatwal) [inline spec/models/teacher_training_adviser/steps/start_teacher_training_spec.rb:190]: Same as above, should this be day 3 ?

**Review by asatwal — APPROVED** [review]
Looks good. I added a few questions about the specs. May need correcting before merging. Thanks.

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for this to be merged

**Pull request merged**